card/spec/models/card/cardtype_spec.rb

```diff
@@ -98,11 +98,11 @@ class Card
       expect(Card['A'].type_code).to eq(:phrase)
     end
   end
-  it "should still have its dependents after changing type" do
+  it "should still have its descendants after changing type" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.